src/operators/IntegralCurve/IntegralCurve.xml

```diff
@@ -72,7 +72,7 @@
       <Enum name="ParallelizationAlgorithmType">
         LoadOnDemand
         ParallelStaticDomains
-        MasterSlave
+        ManagerWorker
```

Comment:
Support needs to be added for existing scripts and saved config/session files. For this class and all others that had attributes renamed.

Reply:
I tend to agree with @biagas, we should maintain backward compatibility for at least one minor release (e.g. until 3.5). We can use the deprecation policy to deprecate it in 3.5. I assume it is only a matter of supporting `Master/Slave` for the parallelization algorithm type.

Reply:
Ok, will do. I'll return to this next week.